libraries/wallet/wallet.cpp

```diff
@@ -770,6 +770,15 @@ class wallet_api_impl
 
       return true;
    }
+
+    void exit_wallet()
+    {
+        ilog( "exiting wallet ..." );
+        save_wallet_file();
+        ilog( "closed wallet successfully" );
+        exit(0);
```

Comment:
i used `exit` before in other parts of the program and @pmconrad recommended to not use it. please check: https://github.com/bitshares/bitshares-core/pull/928#pullrequestreview-120729779

Reply:
I've made changes in commit - just renamed exit_wallet command to quit name. Will check exit and change it too soon, just need to check & test more.

Reply:
you are going to need to send a new pull request or rebase, pull can have only your changes.

Reply:
yes sure, just made mistake, sorry, will fix soon with rebase or new PR

Reply:
fixed already with rebase

Reply:
changed also FC_THROW instead of exit